src/script/api/script_date_economy.hpp

```diff
@@ -5,13 +5,13 @@
  * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
  */
 
-/** @file script_date.hpp Everything to query and manipulate date related information. */
+/** @file script_date_economy.hpp Everything to query and manipulate date related information. */
 
-#ifndef SCRIPT_DATE_HPP
-#define SCRIPT_DATE_HPP
+#ifndef SCRIPT_DATE_ECONOMY_HPP
+#define SCRIPT_DATE_ECONOMY_HPP
 
 #include "script_object.hpp"
-#include "timer/timer_game_calendar.h"
+#include "../../timer/timer_game_economy.h"
 
 /**
  * Class that handles all date related (calculation) functions.
```

Comment:
I would expect some documentation here (and in script_date_calendar.hpp), what is the difference between the two types of dates. Given you, as user, usually only see one I would expect this needs to be clarified for anyone writing an AI or GS.